src/PrettierEditProvider.ts

```diff
@@ -31,9 +31,22 @@ const STYLE_PARSERS: ParserOption[] = ['postcss', 'css', 'less', 'scss'];
  * @param filePath file's path
  */
 async function hasPrettierConfig(filePath: string) {
-    return (await bundledPrettier.resolveConfig(filePath)) !== null;
+    return (await resolveConfig(filePath)) !== null;
 }
 
+/**
+ * Resolves the prettierconfig for the given file.
+ * 
+ * @param filePath file's path
+ */
+async function resolveConfig(filePath: string, options?: { editorconfig?: boolean }): Promise<PrettierConfig | null> {
+    try {
+        return await bundledPrettier.resolveConfig(filePath, options);
+    } catch (e) {
```

Comment:
We shouldn't do that silently
`addToOutput(message: string): void`

Reply:
Done!